### PR TITLE
Add new handlers

### PR DESCRIPTION
**Pull request by Nopock**

### CODE REVIEW

**Comment by Nopock** [comment]
This commit is detremental to all pheonix related stuff and will fix all current dependency errors. 🤓

**Comment by ohemilyy** [comment]
.

**Comment by AndyReckt** [comment]
ratio

**Comment by ohemilyy** [comment]
@CreaxxOG @DevDrizzy

**Comment by CreaxxOG** [comment]
Some of the methods you have added are not present in Phoenix and thus would not work. I will however add the filter stuff.

**Comment by Nopock** [comment]
> Some of the methods you have added are not present in Phoenix and thus would not work. I will however add the filter stuff.

Assuming you are refering to the getProfiles method, it could be as simple as a mongodb query with deserialization (No caching involved) which can be used by a developer. Although this is a heavy task you can leave that for the developer to figure out how to handle it or possibly make it a CompletableFuture if you are up to it.

Thanks,
Nopox

File: src/main/java/xyz/refinedev/phoenix/handler/FilterHandler.java [hunk 1]
```diff
@@ -0,0 +1,20 @@
...
+ * @author Nopox
+ * Created at 9/17/2023
+ */
+@Getter
+public class FilterHandler {
+
+    /**
+     * @param message The message to check.
+     * @return If the message should be filtered.
+     */
```
Comment (CreaxxOG) [inline src/main/java/xyz/refinedev/phoenix/handler/FilterHandler.java:16]: Implemented

**Pull request closed without merging**